packages/operator/pkg/apiserver/routes/v1/deployment/utils.go

```diff
@@ -0,0 +1,29 @@
+package deployment
+
+import (
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/apiserver/routes"
+	"net/http"
+	"strconv"
+)
+
+func ValidateAndParseCursor(c *gin.Context, cursor *int) (ok bool) {
```

Comment:
_Minor:_ if the only purpose of the return value is to indicate a successful/failed parsing, the maybe returning an `error` instead of `bool` would be more idiomatic.